File: test/fixtures/csharp/get_cookie_with_empty_value.cs
using System.Net.Http;

HttpClient client = new HttpClient();

HttpRequestMessage request = new HttpRequestMessage(HttpMethod.Get, "http://localhost:28139/cookies");

request.Headers.Add("accept", "application/json");
request.Headers.Add("Cookie", "mysamplecookie=someValue; emptycookie=; otherCookie=2");

HttpResponseMessage response = await client.SendAsync(request);
response.EnsureSuccessStatusCode();
string responseBody = await response.Content.ReadAsStringAsync();

File: test/fixtures/r/get_cookie_with_empty_value.r
library(httr)

cookies = c(
  mysamplecookie = "someValue",
  emptycookie = "",
  otherCookie = "2"
)

headers = c(
  accept = "application/json"
)

res <- httr::GET(url = "http://localhost:28139/cookies", httr::add_headers(.headers=headers), httr::set_cookies(.cookies = cookies))
